plan/service.py

```diff
@@ -139,21 +140,28 @@ def available_plans(self, owner: Owner) -> List[PlanData]:
         return available_plans
 
     def _start_trial_helper(
-        self, current_owner: Owner, end_date: datetime = None
+        self,
+        current_owner: Owner,
+        end_date: datetime = None,
```

Comment:
This should be datetime | None for the type right?

Reply:
Yes, updated to `end_date: Optional[datetime] = None`